pkg/agent/controller/traceflow/packetin.go

```diff
@@ -236,14 +255,19 @@ func getInfoInTunnelDst(regMatch *ofctrl.MatchField) (string, error) {
 	return regValue.String(), nil
 }
 
-func getInfoInCtNwDstField(matchers *ofctrl.Matchers) (string, error) {
-	match := matchers.GetMatchByName("NXM_NX_CT_NW_DST")
+func getInfoInCTDst(matchers *ofctrl.Matchers, isIPv6 bool) (string, error) {
```

Comment:
Nit: do you think getCTDstValue is better? Again, if you agree to rename it, maybe rename other getInfoXXX funcs to getXXXValue too.